SPECS/mysql/mysql.spec

```diff
@@ -96,6 +98,9 @@ fi
 %{_libdir}/pkgconfig/mysqlclient.pc
 
 %changelog
+* Fri Jan 26 2024 Andy Zaugg <azaugg@linkedin.com> - 8.0.35-4
```

Comment:
Starting a new Buddy Build to build both `mysql & ntopng`:
- [PR-7496](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=492611&view=results)